src/nvapi_d3d.cpp

```diff
@@ -87,4 +88,47 @@ extern "C" {
 
         return Ok(n, alreadyLoggedOk);
     }
+
+    NvAPI_Status __cdecl NvAPI_D3D_Sleep(IUnknown *pDevice) {
+        constexpr auto n = __func__;
+        static bool alreadyLogged = false;
+        if (!nvapiD3dInstance->IsReflexAvailable()) {
+            return NoImplementation(n, alreadyLogged);
+        }
+        nvapiD3dInstance->Sleep();
+        return Ok(n, alreadyLogged);
+    }
+
+    NvAPI_Status __cdecl NvAPI_D3D_SetSleepMode(IUnknown *pDevice,
+                                                NV_SET_SLEEP_MODE_PARAMS *pSetSleepModeParams) {
+        constexpr auto n = __func__;
+        if (pSetSleepModeParams->version != NV_SET_SLEEP_MODE_PARAMS_VER1) return IncompatibleStructVersion(n);
+        if (!nvapiD3dInstance->IsReflexAvailable()) {
+            log::write(str::format(n, ": Wine LFX module unavailable"));
+            return NoImplementation();
+        }
+        auto f = str::format(n, " (", pSetSleepModeParams->bLowLatencyMode ? "Enable" : "Disable", ")");
+        if (pSetSleepModeParams->bLowLatencyMode && !nvapiD3dInstance->IsReflexAvailable()) {
+            log::write(str::format(n, ": tried to enable Reflex but Wine LFX module is not available!"));
+        }
+        nvapiD3dInstance->SetReflexEnabled(pSetSleepModeParams->bLowLatencyMode);
+        if (pSetSleepModeParams->bLowLatencyMode) {
+            log::write(str::format(n, ": interval ", pSetSleepModeParams->minimumIntervalUs, " us"));
+            constexpr uint64_t kNanoInMicro = 1000;
+            nvapiD3dInstance->SetTargetFrameTime(pSetSleepModeParams->minimumIntervalUs * kNanoInMicro);
```

Comment:
I kept the interval logging here. Let me know if that should be moved or removed as well.

Reply:
What about add that info the OK log statement, similar like you did with Enable/Disable?
That said, how often is this method called? Since DXVK-NVAPI unfortunately only knows about info logging, I like to keep the logging limited.